pkg/message/slack.go

```diff
@@ -16,19 +17,22 @@ type message struct {
 
 type Slack struct {
 	client *http.Client
+	notifications []models.NotificationConfig
```

Comment:
We can remap that list into a `map[string][]string` (where `map[LABEL][]CHANNELS`) as maps don't need to be iterated to access an entry. This can be done in `NewSlack` and it will be later used during the whole execution.

Reply:
Good one! 
We may remove the first commit and just use the map. Should be fine to generate it from json, right?

Reply:
Doesn't seems too user-friendly. I rather keep using the channel -> tags at the config.

Reply:
I updated the `helm/values.yaml` with the new format: https://github.com/codelicia/turbo-enigma/pull/14/files#diff-81801117ec01136c8a49bfcd42af8e104f4efad1f2daccda9da9d960eaad5279

What do you think?